Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/ConversationFragment.kt

```diff
@@ -78,9 +64,8 @@ class ConversationFragment : Fragment() {
                         onNavIconPressed = {
                             activityViewModel.openDrawer()
                         },
-                        // Add padding so that we are inset from any left/right navigation bars
-                        // (usually shown when in landscape orientation)
-                        modifier = Modifier.navigationBarsPadding(bottom = false)
+                        // Add padding so that we are inset from any navigation bars
+                        modifier = Modifier.navigationBarsPadding()
```

Comment:
The direct replacement here would be `windowInsetsPadding(WindowInsets.navigationBars.only(WindowInsetsSides.Horizontal + WindowInsetsSides.Top))`